list.go

```diff
@@ -50,29 +50,3 @@ func (list *List) UnmarshalJSON(rawData []byte) error {
 
 	return nil
 }
-
-/*
-MarshalJSON returns a top level object for the "data" attribute if a single object. In
-all other cases returns a JSON encoded list for "data". We use a pointer receiver here
-so we are able to distinguish between nil (don't serialize) and empty (serialize as []).
-*/
-func (list *List) MarshalJSON() ([]byte, error) {
```

Comment:
It appears that due to how we know handle marshaling, we no longer need this custom function. This might just be due to a lack of test coverage. @petercgrant any thoughts? Double checking this is the last step before I think this is GTG.